setup.py

```diff
@@ -77,6 +77,8 @@
     'prettytable'
 ]
 
+if sys.version_info < (3, 7):
+    dependencies[dependencies.index('numpy')] = 'numpy<1.20'
```

Comment:
If we need a limited numpy version anyway, maybe `'numpy<1.19.4'` is better? Due to `'numpy >= 1.19.4'` have compatibility issue in windows. https://tinyurl.com/y3dm3h86